scripts/tcl_commands/routing.tcl

```diff
@@ -408,6 +407,7 @@ proc run_routing {args} {
         }
         if { $::env(RUN_HEURISTIC_DIODE_INSERTION) } {
             heuristic_diode_insertion
+            detailed_placement_or
```

Comment:
Farid told me to leave a comment here idk why

Reply:
I thought we might need detailed placement afterwards but turns out that it is already implemented.